.github/workflows/ci.yml

```diff
@@ -216,6 +218,9 @@ jobs:
         run: |
           Copy-Item -Path "${{ github.workspace }}" -Destination "${{ env.DEV_DRIVE }}/uv" -Recurse
 
+      - name: "Install nasm"
+        uses: ilammy/setup-nasm@v1
```

Comment:
Note to self we should audit this action and consider just implementing it ourself if the install is straightforward

Reply:
ilammy also known for https://github.com/ilammy/msvc-dev-cmd